scripts/apps/authoring/authoring/services/AuthoringService.ts

```diff
@@ -277,16 +277,30 @@ export function AuthoringService($q, $location, api, lock, autosave, confirm, pr
         helpers.filterDefaultValues(extDiff, orig);
         var endpoint = 'archive_' + action;
 
-        return api.update(endpoint, orig, extDiff)
+        return api.update(endpoint, orig, extDiff, {publishing_warnings_confirmed: publishingWarningsConfirmed})
             .then(
                 (result) => lock.unlock(result).catch(() => result), // ignore unlock err
                 (reason) => {
-                    if (notifyErrors && reason != null && get(reason, 'data._issues')) {
+                    const issues = reason?.data?._issues;
+
+                    if (notifyErrors && issues) {
                         Object.values(reason.data._issues).forEach((message) => {
                             if (message != null) {
                                 notify.error(message);
                             }
                         });
+                    } else if (issues?.['validator exception'] && issues?.fields == null) {
+                        const errorObj = JSON.parse(issues?.['validator exception']);
+
+                        if (errorObj.warnings) {
+                            return modal.confirm({
+                                bodyText: errorObj.warnings.toString(),
```

Comment:
Why are you casting warnings to string? Can you add a type for `errorObj`?

Does it support displaying multiple messages properly? If not, you can use [`showModal`](https://github.com/superdesk/superdesk-client-core/blob/develop/scripts/core/services/modalService.tsx#L7) function with a custom React component to do this.